node/src/reactor/queue_kind.rs

```diff
@@ -30,6 +30,18 @@ pub enum QueueKind {
     Api,
 }
 
+impl Display for QueueKind {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let str_value = match self {
+            QueueKind::NetworkIncoming => "NetworkIncoming",
```

Comment:
I think our convention for metrics is to `snake_case` them, if I am not mistaken? So this would be `"network_incoming"`?

Reply:
I didn't want the `Display` impl be tailored for Prometheus metrics. It's fine if it's this way, it won't break prometheus.

Reply:
Well, this way it is exactly the same - I guess we should make it a `fn metrics_name(&sefl)` then instead, if that's a concern?

Reply:
Done. Added `metrics_name`.